SPECS/python-toolz/python-toolz.spec

```diff
@@ -61,15 +64,19 @@ BuildRequires:  python%{python3_pkgversion}-pip
 %py3_install
 
 %check
-pip3 install nose
-nosetests
+# skip test_shakespeare due to missing directory/files in source tar (Fedora also skips this test for a similar reason)
```

Comment:
I think it is a good idea to credit Fedora in the changelog if you inherited the patch or any idea from Fedora.

Reply:
Sure I will tag accordingly